buildConfiguration.xml

```diff
@@ -2,7 +2,7 @@
   <dotnetArchitecture>x64</dotnetArchitecture>
   <nugetVersion>3.5.0-rc-1285</nugetVersion>
   <runtimes>net461,netstandard2.0</runtimes>
-  <assemblyVersion>7.1.2</assemblyVersion>
+  <assemblyVersion>7.2.0</assemblyVersion>
```

Comment:
we already have this version shipped; https://www.nuget.org/packages/Microsoft.IdentityModel.Tokens/7.2.0

should be 7.3.0 for this release ?